app/client/src/utils/hooks/cyanWaterbodies.ts

```diff
@@ -0,0 +1,330 @@
+import Color from '@arcgis/core/Color';
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+import Graphic from '@arcgis/core/Graphic';
+import GroupLayer from '@arcgis/core/layers/GroupLayer';
+import MediaLayer from '@arcgis/core/layers/MediaLayer';
+import Polygon from '@arcgis/core/geometry/Polygon';
+import * as query from '@arcgis/core/rest/query';
+import SimpleFillSymbol from '@arcgis/core/symbols/SimpleFillSymbol';
+import SimpleRenderer from '@arcgis/core/renderers/SimpleRenderer';
+import { useCallback, useContext, useEffect, useRef, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+// contexts
+import { useFetchedDataDispatch } from 'contexts/FetchedData';
+import { LocationSearchContext } from 'contexts/locationSearch';
+import { useServicesContext } from 'contexts/LookupFiles';
+// utils
+import {
```

Comment:
```suggestion
// utils
import { useDynamicPopup } from 'utils/hooks';
import {
```

This is related to getting the `Change to this location` section in the surrounding features popups. 

This and other changes below will need to be done in the `dischargers.ts`, `monitoringLocations.tsx`, and `streamgages.ts` files.

Reply:
I knew something was off with those popups! Nice catch, I'll make those changes for all surrounding layers.